packages/backend/src/plugins/search.ts

```diff
@@ -22,13 +22,20 @@ import {
 import { PluginEnvironment } from '../types';
 import { DefaultCatalogCollator } from '@backstage/plugin-catalog-backend';
 import { DefaultTechDocsCollator } from '@backstage/plugin-techdocs-backend';
+import { ElasticSearchSearchEngine } from '@backstage/plugin-search-backend-module-elasticsearch';
 
 export default async function createPlugin({
   logger,
   discovery,
+  config,
 }: PluginEnvironment) {
   // Initialize a connection to a search engine.
-  const searchEngine = new LunrSearchEngine({ logger });
+  const searchEngine = config.has('search.elasticSearch')
```

Comment:
@iamEAP should this be added to the `create-app` template or should we just have this on docs?

Reply:
I'm good w/just docs.  The value here is making it easy to switch in the context of the example backend.  In a real instance, it'd just be instantiating the real thing.